### PR TITLE
submit latest version to ios testflight

### PR DESCRIPTION
**Pull request by bestw07**

### CODE REVIEW

**Comment by TreeMama** [comment]
Three big bugs so far #496, #497 and #495

File: iOS/iSeaTree.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -361,7 +361,7 @@
 				CODE_SIGN_IDENTITY = "Apple Development";
 				CODE_SIGN_STYLE = Automatic;
 				CURRENT_PROJECT_VERSION = 1;
-				DEVELOPMENT_TEAM = "";
+				DEVELOPMENT_TEAM = 56U5Z4MD27;
```
Comment (TreeMama) [inline iOS/iSeaTree.xcodeproj/project.pbxproj:364]: @bestw07 is this ok to show in git?

Comment (bestw07) [reply iOS/iSeaTree.xcodeproj/project.pbxproj:364]: > @bestw07 is this ok to show in git?

that's no problem. it's team ID.

**Review by TreeMama — APPROVED** [review]
thanks @bestw07 - @dceu it looks like your branch was not sync'd to the latest develop when we approved your PR. Mony fixed it though (thanks @bestw07 ).

**Pull request merged**